bitcoin-ticker/bitcoinTicker.js

```diff
@@ -1,6 +1,32 @@
+const http = require('https');
 
-function bitcoinTicker(...args) {
-  return args;
+const options = {
+  hostname: 'api.coinmarketcap.com',
+  path: '/v2/ticker/',
+  method: 'GET',
+};
+
+function bitcoinTicker({ limit }) {
+  return new Promise((resolve) => {
+    http.get(options, (res) => {
```

Comment:
Doesn't http.get implicitly specifies that method is get? Do we need to supply an object with property method?